src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -643,13 +643,28 @@ public virtual void Generate(NpgsqlDropDatabaseOperation operation, [CanBeNull]
 
             GenerateEnumStatements(operation, model, builder);
             GenerateRangeStatements(operation, model, builder);
-
-            foreach (var extension in operation.Npgsql().PostgresExtensions)
-                GenerateCreateExtension(extension, model, builder);
+            GenerateExtensionStatements(operation, model, builder);
```

Comment:
While this isn't strictly necessary, I would like to bring extension handling into line with enums and ranges. 

Even if we don't end up supporting drop/alter operations for extensions in 3.0, I think there's value in standardizing the API at this point.